README.md

```diff
@@ -1,67 +1,146 @@
+- Information on famous bald people
+
+## Installation
+
+Requires [NodeJS v16.x.x](https://nodejs.org/en/) or higher and the latest version of [NPM](https://docs.npmjs.com/cli/v7/commands/npm-install)
+
+### Windows
+
+```
+$ windows_setup.bat
+```
+
+### Manual
+
+In package directory run
+
+```
+$ npm install
+```
+
+## Setup
+
+In the repo folder create a file called ``` .env ``` with the contents
+
+```
+DISCORD_TOKEN=your-discord-bot-token-goes-here
+CLIENT_ID=your-discord-client-id-goes-here
+```
+
+To get a Discord bot token and Discord client ID see [How to Make a Discord Bot in the Developer Portal](https://realpython.com/how-to-make-a-discord-bot-python/#how-to-make-a-discord-bot-in-the-developer-portal).
+
+We utilize [say.js](https://github.com/Marak/say.js) to convert text strings to mp3 files. Say.js uses some [FFMpeg](https://www.ffmpeg.org/) libraries to operate, so you will need to install the most up to date version. You can learn how to do that [here](https://www.wikihow.com/Install-FFmpeg-on-Windows). 
+
+(Note: say.js uses [Festival](https://www.cstr.ed.ac.uk/projects/festival/) to operate on Linux, so check their [README](https://github.com/Marak/say.js/blob/master/README.md) to get that squared away)
+
+If you want to utilize Pastrami's audio streaming capabilities for non YouTube content, you will need to populate ``` /data/audiofiles``` with the mp3/mp4 files you would like to play.
+
+## Usage
+
+### Startup 
+
+To start the bot locally on your machine run
+
+```
+$ node index.js
+```
+
+This will also connect the bot to any guilds you have linked it to via [Discord's Developer Portal](http://discordapp.com/developers/applications).
+
+### Greetings
+
+To customize greetings navigate to ``` /data/greetings.json```. 
+
+- Default Greetings
+
+```json
+"default": {
+        "greetings": [
+            "welcome *NAME*",
+            "say hi to *NAME*"
+        ]
+    },
+```
+
+In ``` greetings.json```  the \*NAME\* variable is used to reference the server nickname of the user who has joined the voice channel. 
+
+- Guild Default Greetings 
+
+```json
+"xxxx": {
+        "greetings": [
+            "It's *NAME*!",
+            "*NAME* has arrived"
+        ]
+    },
+```
+
+This is pretty much the same as the default greetings, but it requires the unique guild identifier ( ``` xxxx``` ) to function. 
+
+- User Specific Greetings
+
+```json
+"xxxx": {
+        "greetings": [
+            "Malachi is cool",
+            "Everyone likes Malachi"
+        ]
+    },
+```
+
+Again, this is pretty similar to the previous greeting formats. However, for user specific greetings you need to replace ``` xxxx``` with that user's unique identifier. 
+
+*To find the unique identifiers for guilds and users check [this](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) out.*
+
+(Note: Greetings within the same grouping will be used randomly with equal probability)
+
+### Slash Commands

```

Comment:
great work listing all of the commands and their description. If you're up to it, you could format this information using a table like here: https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables - this would add a lot more readability imo